plugins/prestashop/prestashop_

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+: << =cut
+=head1 NAME
+
+Munin Plugin to grab PrestaShop statistics
+
+=head1 CONFIGURATION
+
+# Sample:
+[prestashop]
+	user root
+	env.database prestashop
+	env.dbprefix ps_
+
+=head1 AUTHOR
+
+# (c) 2015 Raphaël Droz <raphael.droz+floss@gmail.com>
+# General Public Licence v3 or later
+
+=head1 LICENSE
+
+GPLv3
+
+=head1 MAGIC MARKERS
+
+#%# family=auto
+#%# capabilities=autoconf suggest
+
+=cut
+
+graph_type=${0##*_}
+
+
+if [[ $1 = autoconf ]]; then
+    let ok=1
+    [[ -z $database ]] && echo "no (env.database not defined)" && ok=0
+    [[ -z $dbprefix ]] && echo "no (env.dbprefix not defined)" && ok=0
+    [[ -z $graph_type ]] && echo "no plugins linked with a bad name (unknown graph type '$graph_type')" && ok=0
```

Comment:
The descriptive text for "no" feels hard to read and does not seem to reflect the test causing it. Maybe just pick `no (invalid link or missing graph type)`?